src/solidity.ts

```diff
@@ -191,6 +205,27 @@ abstract class SolidityContractItem implements Linkable {
   }
 }
 
+class SolidityVariable extends SolidityContractItem {
+  constructor(
+    contract: SolidityContract,
+    protected readonly astNode: solc.ast.VariableDeclaration,
+  ) {
+    super(contract, astNode);
+  }
+
+  get type(): string {
+    return this.astNode.typeName.typeDescriptions.typeString;
+  }
+
+  get signature(): string {
```

Comment:
Please remove this function, for now we will not expose the signature of the functions generated by variables. (See the comment above.)

Reply:
I believe that this function is actually very much relevant for variables.
First, because on the philosophical aspect, a variable-declaration can be considered a signature just as much as a function-declaration can.
Second, because on the technical aspect, it is used by the anchor function, which I think we should support for variables.